src/main/java/org/terracotta/offheapstore/OffHeapHashMap.java

```diff
@@ -1445,6 +1447,10 @@ private void wipePendingTables() {
       }
     }
   }
+
+  protected K keyForEntry(IntBuffer entry) {
```

Comment:
The storage engine is already protected so this new method isn't strictly required.  I'm not anti the approach here... but this feels like a one part of a changeset with a wider purpose... if we wanted to refactor some of this stuff with the aim of cleaning up then I would be in support but given everything else is just changing modifiers this feels weird.